cmd/govim/testdata/show_message.txt

```diff
@@ -0,0 +1,19 @@
+# Test that popup shows up when message is of Error type
+
+[!vim] [!gvim] skip
+
+# For some reason, the new-style popup test fails on gvim.
+# https://github.com/govim/govim/issues/351
+[!vim:v8.1.1649] skip
+
+vim expr 'GOVIM_internal_ShowMessagePopup()'
+cmp stdout showMessage.golden
```

Comment:
This is really superfluous for this test. We have tests for function return values elsewhere